### DISCUSSION TITLE
Debug with Directory feedback on Intro Screen

### DISCUSSION THREAD

**Discussion by bradhannah** [bradhannah/Ultima5Redux#332] [Show and tell]
For those who have struggled with the directory structures (because they are a mystery!!!) I have slapped some debugging on the main menu to hopefully help.

![image](https://user-images.githubusercontent.com/47454550/184264894-fa273449-664b-4573-8970-5ea436ae3aff.png)

**Comment by bradhannah** [comment]
@tanabi

**Comment by tanabi** [comment]
I'll give this a try this weekend and see what it's like :)

**Comment by bradhannah** [comment]
I’ll need to get a new build up first, will try to do sooner than later.
